### ISSUE TITLE
[Question]: Ernie-Layout模型代码跟论文不一致

### ISSUE THREAD

**Issue by will-wiki** [PaddlePaddle/PaddleNLP#4573]
### 请提出你的问题

你好，最近阅读了Ernie-Layout模型的代码和论文，有点问题想请教下。
1.github中model_zoo里开源的代码是不是只有finetune的代码，跑了funsd的训练代码，只有构造BIO的训练loss，而论文里是有四个预训练任务的。
2.代码中transformer的自注意力机制跟论文描述的不一样，原版是没有开源吗？
3.代码里没有发现相关Serialization Module部分的代码，这个部分是不开源吗？

**Comment by linjieccc** [comment]
你好，感谢对ERNIE-Layout的关注！

1）Serialization module在ERNIE-Layout的预训练阶段有使用，预训练代码暂不开源
2）参考 #3489

**Comment by will-wiki** [comment]
@linjieccc 你好，想问下ernie-layout有相关中文预训练模型吗，官方文档里面只看到ERNIE-LayoutX-Base，这个应该是英文的吧

**Comment by linjieccc** [comment]
@will-wiki ERNIE-LayoutX-Base是在中英文语料上训练的，因为用的是跨语言底座ERNIE-M，所以具备96种语言的能力，可以通过[HF demo](https://huggingface.co/spaces/PaddlePaddle/ERNIE-Layout)体验

**Comment by will-wiki** [comment]
@linjieccc 想问下有专门针对中文的预训练模型开源吗

**Comment by will-wiki** [comment]
@linjieccc 你好，想问下ernie-layout有专门针对中文的预训练模型开源吗

**Comment by will-wiki** [comment]
@linjieccc 想问下有人吗？能否解答下上面的问题

**Comment by linjieccc** [comment]
不好意思 漏消息，抱歉！

目前没有专门针对中文的，可以以ERNIE-Layout为底座，效果上不会弱于同样尺寸的纯中文预训练模型

**Comment by will-wiki** [comment]
@linjieccc 你上面说的“以ERNIE-Layout为底座“是什么意思呢，https://github.com/PaddlePaddle/PaddleNLP/blob/develop/model_zoo/ernie-layout/README_ch.md官方的说明文档里只有提到ERNIE-LayoutX-Base模型，这里的意思是还有一个ERNIE-Layout模型吗

**Comment by linjieccc** [comment]
就是基于ERNIE-Layout做fine-tuning，文档里提供了三大任务的微调示例（分类、NER、阅读理解），Base指的是模型大小

**Comment by will-wiki** [comment]
@linjieccc 想问下ernie-layout支持分布式训练吗，用如下方法调用现在的训练脚本就可以吗？
python -m paddle.distributed.launch  --gpus="0,1" run_ner.py \
  --model_name_or_path ernie-layoutx-base-uncased \
  --output_dir ./ernie-layoutx-base-uncased/models/xfund_zh/ \
  --dataset_name xfund_zh \
  --do_train \
  --do_eval \
  --lang "ch" \
  --max_steps 20000 \
  --eval_steps 100 \
  --save_steps 100 \
  --save_total_limit 1 \
  --load_best_model_at_end \
  --pattern ner-bio \
  --preprocessing_num_workers 4 \
  --overwrite_cache false \
  --use_segment_box \
  --doc_stride 128 \
  --target_size 1000 \
  --per_device_train_batch_size 4 \
  --per_device_eval_batch_size 4 \
  --learning_rate 1e-5 \
  --lr_scheduler_type constant \
  --gradient_accumulation_steps 1 \
  --seed 1000 \
  --metric_for_best_model eval_f1 \
  --greater_is_better true \
  --overwrite_output_dir
